On loongarch64 and riscv64, the internal/cmd tests timeout. Increase the timeout
to give those arches a chance to pass.

diff --git a/Makefile b/Makefile
index 421bd9d..6f0ea27 100644
--- a/Makefile
+++ b/Makefile
@@ -91,8 +91,8 @@ rm-dist:
 
 .PHONY: test
 test:
-	${GO} test -ldflags="-X github.com/twpayne/chezmoi/v2/internal/chezmoitest.umaskStr=0o022" ./...
-	${GO} test -ldflags="-X github.com/twpayne/chezmoi/v2/internal/chezmoitest.umaskStr=0o002" ./...
+	${GO} test -timeout 20m -ldflags="-X github.com/twpayne/chezmoi/v2/internal/chezmoitest.umaskStr=0o022" ./...
+	${GO} test -timeout 20m -ldflags="-X github.com/twpayne/chezmoi/v2/internal/chezmoitest.umaskStr=0o002" ./...
 
 .PHONY: test-docker
 test-docker:
